gtsam/hybrid/HybridFactorGraph.h

```diff
@@ -31,7 +31,7 @@ using SharedFactor = boost::shared_ptr<Factor>;
  * NonlinearFactorGraph, GaussianFactorGraph.
  */
 template <typename FG>
-class HybridFactorGraph : protected FactorGraph<Factor> {
+class HybridFactorGraph : public FactorGraph<Factor> {
```

Comment:
This may have some implications (exposing some stuff that's not suitable for the Hybrid factor graph) but we'll see

Reply:
Yeah I thought about that. However, everywhere else the inheritance is public. HybridFactorGraph is the only place it is protected.